From 04763135d47ae24a808fc15c4482e2bb6f847ab9 Mon Sep 17 00:00:00 2001
From: Akira TAGOH <akira@tagoh.org>
Date: Wed, 25 Nov 2015 11:58:14 +0900
Subject: Avoid an error message on testing when no fonts.conf installed

This test case doesn't require any config files so no need to ensure loading them.

diff --git a/test/test-bz89617.c b/test/test-bz89617.c
index 587214a..35df885 100644
--- a/test/test-bz89617.c
+++ b/test/test-bz89617.c
@@ -28,7 +28,7 @@
 int
 main (void)
 {
-    FcConfig *config = FcConfigGetCurrent ();
+    FcConfig *config = FcConfigCreate ();
 
     if (!FcConfigAppFontAddFile (config, (const FcChar8 *)SRCDIR "/4x6.pcf") ||
 	FcConfigAppFontAddFile (config, (const FcChar8 *)"/dev/null"))
-- 
cgit v0.10.2

